workshop_sections/word2vec/README.md

```diff
@@ -1,120 +1,178 @@
 # Building a Custom Estimator for word2vec
```

Comment:
I'd add a bit more at the top of the readme as to what 'word2vec' is (maybe a link etc.)